info: Conqueror.Middleware.Logging.Tests.Signalling.LoggingMiddlewareTestSignals.TestSignalWithCustomJsonTypeInfo[441733974]
      Publishing in-process signal of type 'TestSignalWithCustomJsonTypeInfo' with payload
      {
        "SIGNAL_PAYLOAD": 10
      }
      (Signal ID: test-message-id, Trace ID: test-trace-id)
info: Conqueror.Middleware.Logging.Tests.Signalling.LoggingMiddlewareTestSignals.TestSignalWithCustomJsonTypeInfoHandler[441733974]
      Handling signal of type 'TestSignalWithCustomJsonTypeInfo' with payload
      {
        "SIGNAL_PAYLOAD": 10
      }
      (Signal ID: test-message-id, Trace ID: test-trace-id)
fail: Conqueror.Middleware.Logging.Tests.Signalling.LoggingMiddlewareTestSignals.TestSignalWithCustomJsonTypeInfoHandler[712209902]
      An exception occurred while handling signal of type 'TestSignalWithCustomJsonTypeInfo' after 123.4560ms (Signal ID: test-message-id, Trace ID: test-trace-id)
      Conqueror.Middleware.Logging.Tests.Signalling.LoggingMiddlewareTestSignals+TestException: test exception
         at Conqueror.Middleware.Logging.Tests.Signalling.LoggingMiddlewareTestSignals.TestSignalWithCustomJsonTypeInfoHandler.Handle(TestSignalWithCustomJsonTypeInfo signal, CancellationToken cancellationToken) in {ProjectDirectory}Signalling/LoggingMiddlewareTestSignals.cs:line FIXED
         at Conqueror.Middleware.Logging.Signalling.LoggingSignalMiddleware`1.Execute(SignalMiddlewareContext`1 ctx) in {SolutionDirectory}src/Conqueror.Middleware.Logging/Signalling/LoggingSignalMiddleware.cs:line FIXED
         at System.Runtime.CompilerServices.AsyncMethodBuilderCore.Start[TStateMachine](TStateMachine& stateMachine)
         at Conqueror.Middleware.Logging.Signalling.LoggingSignalMiddleware`1.Execute(SignalMiddlewareContext`1 ctx)
         at Conqueror.Signalling.SignalPipelineRunner`1.<>c__DisplayClass3_2.<Execute>b__1(TSignal signal, CancellationToken token) in {SolutionDirectory}src/Conqueror/Signalling/SignalPipelineRunner.cs:line FIXED

fail: Conqueror.Middleware.Logging.Tests.Signalling.LoggingMiddlewareTestSignals.TestSignalWithCustomJsonTypeInfo[712209902]
      An exception occurred while publishing in-process signal of type 'TestSignalWithCustomJsonTypeInfo' after 123.4560ms (Signal ID: test-message-id, Trace ID: test-trace-id)
      Conqueror.Middleware.Logging.Tests.Signalling.LoggingMiddlewareTestSignals+TestException: test exception
         at Conqueror.Middleware.Logging.Tests.Signalling.LoggingMiddlewareTestSignals.TestSignalWithCustomJsonTypeInfoHandler.Handle(TestSignalWithCustomJsonTypeInfo signal, CancellationToken cancellationToken) in {ProjectDirectory}Signalling/LoggingMiddlewareTestSignals.cs:line FIXED
         at Conqueror.Middleware.Logging.Signalling.LoggingSignalMiddleware`1.Execute(SignalMiddlewareContext`1 ctx) in {SolutionDirectory}src/Conqueror.Middleware.Logging/Signalling/LoggingSignalMiddleware.cs:line FIXED
         at Conqueror.Signalling.SignalPipelineRunner`1.Execute(IServiceProvider serviceProvider, TSignal initialSignal, ISignalPublisher`1 publisher, SignalTransportType transportType, CancellationToken cancellationToken) in {SolutionDirectory}src/Conqueror/Signalling/SignalPipelineRunner.cs:line FIXED
         at Conqueror.Signalling.SignalDispatcher`1.Dispatch(TSignal signal, CancellationToken cancellationToken) in {SolutionDirectory}src/Conqueror/Signalling/SignalDispatcher.cs:line FIXED
         at Conqueror.Signalling.SequentialSignalBroadcastingStrategy.BroadcastSignal[TSignal](IReadOnlyCollection`1 signalHandlerInvocationFns, IServiceProvider serviceProvider, TSignal signal, CancellationToken cancellationToken) in {SolutionDirectory}src/Conqueror/Signalling/SequentialSignalBroadcastingStrategy.cs:line FIXED

